src/coreComponents/fileIO/Outputs/BlueprintOutput.cpp

```diff
@@ -68,6 +68,11 @@ static std::vector< int > getBlueprintNodeOrdering( ElementType const elementTyp
     case ElementType::Hexahedron:    return { 0, 1, 3, 2, 4, 5, 7, 6 };
     case ElementType::Prism5:        return { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }; // TODO
     case ElementType::Prism6:        return { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }; // TODO
+    case ElementType::Prism7:        return { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }; // TODO
+    case ElementType::Prism8:        return { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }; // TODO
+    case ElementType::Prism9:        return { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }; // TODO
+    case ElementType::Prism10:       return { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }; // TODO
+    case ElementType::Prism11:       return { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 }; // TODO
```

Comment:
If you want to return dummy values, do you prefer them to be _empty_ (`{}`) or full of zeros with the correct size?
Because you've added entries with the wrong number of zeros.